services/brig/test/integration/API/User/Handles.hs

```diff
@@ -335,7 +335,8 @@ assertCanFind brig from target = do
   let targetHandle = fromMaybe (error "Impossible") (userHandle target)
   get (apiVersion "v1" . brig . path "/users" . queryItem "handles" (toByteString' targetHandle) . zUser (userId from)) !!! do
     const 200 === statusCode
-    const (userHandle target) === (>>= (listToMaybe >=> profileHandle)) . responseJsonMaybe
+    const (userHandle target) === ((listToMaybe >=> profileHandle) <=< responseJsonMaybe)
```

Comment:
```suggestion
    const (userHandle target) === (responseJsonMaybe >=> listToMaybe >=> profileHandle)
```

(not tested)

Reply:
I'm now surprised the linter didn't suggest this itself… I'll test it in a bit.